superset-frontend/src/explore/components/ExploreChartPanel.jsx

```diff
@@ -49,22 +52,124 @@ const propTypes = {
   triggerRender: PropTypes.bool,
 };
 
+export const EXPLORE_GUTTER_HEIGHT = 5;
+export const EXPLORE_GUTTER_MARGIN = 3;
+export const CHART_PANEL_PADDING = 30;
+
+const INITIAL_SIZES = [90, 10];
+const MIN_SIZES = [300, 50];
+
 const Styles = styled.div`
   background-color: ${({ theme }) => theme.colors.grayscale.light5};
   height: 100%;
   display: flex;
   flex-direction: column;
   align-items: stretch;
   align-content: stretch;
+  overflow: hidden;
+
   & > div:last-of-type {
     flex-basis: 100%;
   }
+
+  .gutter {
+    border-top: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+    border-bottom: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+    width: 3%;
```

Comment:
should the gutter be a % of the total width? Seems like a constant size would be better (both for folks with small and large width monitors)

Reply:
`gridUnit` FTW! :)

Reply:
Done. I set it to gridUnit * 9, which is give or take a width that we used on full hd screens